test/functional/test_framework/mininode.py

```diff
@@ -35,18 +35,45 @@
 
 logger = logging.getLogger("TestFramework.mininode")
 
-# Keep our own socket map for asyncore, so that we can track disconnects
-# ourselves (to workaround an issue with closing an asyncore socket when
-# using select)
-mininode_socket_map = dict()
-
-# One lock for synchronizing all data access between the networking thread (see
-# NetworkThread below) and the thread running the test logic.  For simplicity,
-# NodeConn acquires this lock whenever delivering a message to a NodeConnCB,
-# and whenever adding anything to the send buffer (in send_message()).  This
-# lock should be acquired in the thread running the test logic to synchronize
-# access to any data shared with the NodeConnCB or NodeConn.
-mininode_lock = threading.RLock()
+MESSAGEMAP = {
```

Comment:
nit: this was re-ordered alphabetically in the original PR, should probably do the same to minimize potential merge conflicts in the future

Reply:
https://github.com/dashpay/dash/pull/3372/commits/f7ae32703ae0e80f85f63945ed99914eff8cb65f